.github/workflows/cre-system-tests.yaml

```diff
@@ -125,6 +126,7 @@ jobs:
           go install
           github.com/smartcontractkit/chainlink-testing-framework/tools/gotestloghelper@v1.1.1
 
+      # TODO: checkout that repo and build each capability instead
```

Comment:
Valid?

Reply:
will be done in a separate PR, don't want to make this huger than it already is